src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -88,7 +88,7 @@ public GameType getGameType() {
     @Override
     public String getInvitationCode() {
         if (invitationCode == null) {
-            this.invitationCode = InvitationCodeGenerator.getAlphanumeric();
+            this.invitationCode = InvitationCodeGenerator.getAlphanumericIdCode(getId());
```

Comment:
can you inline "this.id" instead of "getId()"?
That's how, we save one method call on the call stack, "getId()" does nothing but returning "this.id" afaik.

Reply:
true yes!